### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: "Link", "Refresh" button  are not accessible with keyboard in Run\debug configurations window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1438]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select "edit configurations" present at right side of the dialog.
4. "Run/Debug Configurations" dialog will open.
5. Navigate and try to access the  "Link", "Refresh" button with keyboard.

**Expected Result:**
"Link", "Refresh" button should be accessible with keyboard in Run\debug configurations window.

**Actual Result:**
"Link", "Refresh" button  are not accessible with keyboard in Run\debug configurations window.

**Note**: User able to access the buttons by using below keys:
Before Launch -'Alt+b'
Add- 'Alt+insert'
Remove- 'Alt+Delete'
Edit- 'Enter'
Up- 'Alt+Up'
Down- 'Alt+down'.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![buttons not accessible in rundebug config window](https://user-images.githubusercontent.com/22956130/35727831-58a65632-07be-11e8-8854-52885ae4223c.png)

**Comment by jdneo** [comment]
Reference: https://youtrack.jetbrains.com/issue/IDEA-191322

**Comment by Eskibear** [comment]
External issues tracked in #1627

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. Once the dialog is open, it automatically refresh the list, so the feature is hardly ever used. About the web.config file, it contains web app's properties, mostly user don't have to pay attention to it.
* **Quality of experience**: not impacted. A simple workaround for "Refresh": the user can re-open the dialog, it refreshes the list automatically.  And the workaround for the link: the user can check the config file manually in the Azure portal.
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by WiproPSAccTest** [comment]
Agreed with the resolution , as there is a work around that user can check the config file manually in the Azure portal.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749902
